content/id/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Batas Rentang (Limit Range)
+content_template: templates/concept
+weight: 10
+---
+
+<!-- overview -->
+
+Secara bawaan, Container berjalan dengan [sumber daya komputasi](id/docs/user-guide/compute-resources) tanpa batas pada klaster Kubernetes.
+Dengan kuota sumber daya, administrator klaster dapat membatasi konsumsi dan pembuatan sumber daya berbasis {{<glossary_tooltip text="namespace" term_id="namespace">}}.
+Di dalam _namespace_, Pod atau Container dapat mengkonsumsi CPU dan memori sesuai dengan yang ditentukan oleh kuota sumber daya _namespace_. Ada kekhawatiran bahwa satu Pod atau Container dapat memonopoli semua sumber daya yang tersedia. LimitRange adalah kebijakan untuk membatasi alokasi sumber daya (bagi Pod atau Container) pada _namespace_.
+
+
+<!-- body -->
+
+LimitRange memberikan batasan yang dapat:
+
+- Menerapkan penggunaan sumber daya komputasi minimum dan maksimum untuk setiap Pod atau Container dalam _namespace_.
+- Menerapkan permintaan tempat penyimpanan minimum dan maksimum untuk setiap PersistentVolumeClaim dalam _namespace_.
+- Menerapkan rasio antara permintaan dan batas untuk sumber daya dalam _namespace_.
+- Menetapkan permintaan/batas bawaan untuk menghitung sumber daya dalam _namespace_ dan memasukkannya secara otomatis ke Container pada _runtime_.
+
+## Mengaktifkan LimitRange
+
+Dukungan LimitRange diaktifkan secara bawaan untuk banyak distribusi Kubernetes. Hal ini
+diaktifkan ketika tanda `--enable-admission-plugins=` pada apiserver memiliki admission controller `LimitRanger` sebagai
+salah satu argumennya.
+
+LimitRange diberlakukan pada _namespace_ tertentu ketika ada sebuah objek LimitRange pada _namespace_ tersebut.
+
+Nama dari objek LimitRange harus merupakan sebuah [nama subdomain DNS](id/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
+
+### Gambaran Umum LimitRange
+
+- Administrator membuat sebuah LimitRange dalam sebuah _namespace_.
+- Pengguna membuat sumber daya seperti Pod, Container, dan PersistentVolumeClaims pada namespace.
+- _Admission controller_ `LimitRanger` memberlakukan bawaan dan batas untuk semua Pod dan Container yang tidak menetapkan persyaratan sumber daya komputasi dan melacak penggunaannya untuk memastikan agar tidak melebihi minimum, maksimum dan rasio sumber daya yang ditentukan dalam LimitRange yang ada pada _namespace_.
+- Apabila permintaan membuat atau memperbarui sumber daya (Pod, Container, PersistentVolumeClaim) yang melanggar batasan LimitRange, maka permintaan ke server API akan gagal dengan kode status HTTP `403 FORBIDDEN` dan sebuah pesan yang menjelaskan batasan yang telah dilanggar.
```

Comment:
server API or API server? Make it consistent, both are actually valid
```suggestion
- Apabila permintaan membuat atau memperbarui sumber daya (Pod, Container, PersistentVolumeClaim) yang melanggar batasan LimitRange, maka permintaan ke API server akan gagal dengan kode status HTTP `403 FORBIDDEN` dan sebuah pesan yang menjelaskan batasan yang telah dilanggar.
```

Reply:
I prefer `server API`.